### DISCUSSION TITLE
Recover Tik Tok accounts

### DISCUSSION THREAD

**Discussion by louay00** [community/community#43376] [Other Feature Feedback, Questions, & Ideas]
Hello my friends, can someone help me? I want a way to recover blocked tiktok accounts. Do I find a solution for anyone?

**Comment by ihnatenkoo** [comment]
Hello
Contacting TikTok about banned accounts

Accounts that consistently violate [Community Guidelines](https://www.tiktok.com/community-guidelines?lang=en) will be banned from TikTok.

If your account has been banned, you will receive a banner notification when you next open the app, informing you of this account change.

If you believe your account was banned incorrectly, let us know by submitting an appeal.

To submit an appeal:
1. Open the notification.
2. Tap Appeal.
3. Follow the instructions provided.

**Reply by louay00** [reply]
I tried this and it didn't work. I heard that there are recovery methods via paid links

**Comment by queenofcorgis** [comment]
Hi @louay00 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on TikTok like the post above. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by louay00** [reply]
Thank you for the reply. Honestly, I'm new here and I don't know how things go. This is why I'm posting here and I didn't know it was the wrong place😱
